nouveau/src/main/java/org/apache/couchdb/nouveau/NouveauApplicationConfiguration.java

```diff
@@ -24,7 +24,7 @@ public class NouveauApplicationConfiguration extends Configuration {
     @Min(10)
     private int maxIndexesOpen = 10;
 
-    @Min(10)
+    @Min(5)
```

Comment:
does this have to match the integer in the next line?

Reply:
No, it doesn't. 10 is the default, but @Min prevented me setting it lower than. I made this change to speed up testing as nouveau peer checkpoints only advance after the commit